LedService/app/src/main/aidl/com/bksnake/ledservice/ILedControllerInterface.aidl

```diff
@@ -4,16 +4,23 @@ package com.bksnake.ledservice;
 // Declare any non-default types here with import statements
 
 interface ILedControllerInterface {
+
+    /**
+     * Led state instead boolean
+     */
+    const int STATE_OFF = 0;
```

Comment:
Not so good practice to define constants in the interface. 
It will work? but it is anti pattern.

Reply:
It is work. Could  you comment it https://android.googlesource.com/platform/frameworks/av/+/master/camera/aidl/android/hardware/ICameraService.aidl ? 

They use cost variable inside interface.

Reply:
Sorry, I made a mistake. "," -> "?"
I wanted to say that it will work, but it is an anti-pattern. It is question of religion.

Reply:
@AksonovSergei So I need remove STATE_OFF or STATE_ON, right? Ok I will remove it.

Reply:
It is up to you.

Reply:
@AksonovSergei Done